src/scene/materials/standard-material.js

```diff
@@ -1172,13 +1160,16 @@ var _defineMaterialProps = function (obj) {
     _defineAlias(obj, "opacityVertexColor", "opacityMapVertexColor");
     _defineAlias(obj, "lightVertexColor", "lightMapVertexColor");
 
-    for (var i = 0; i < _propsSerial.length; i++) {
+    for (let i = 0; i < _propsSerial.length; i++) {
         _propsSerialDefaultVal[i] = obj[_propsSerial[i]];
     }
 
     obj._propsSet = [];
-};
+}
 
 _defineMaterialProps(StandardMaterial.prototype);
 
+StandardMaterial.TEXTURE_PARAMETERS = standardMaterialTextureParameters;
+StandardMaterial.CUBEMAP_PARAMETERS = standardMaterialCubemapParameters;
```

Comment:
Declare as static in the StandardMaterial class?

Reply:
Done